### PR TITLE
Reorganised documentation and added RBAC section

### PR DESCRIPTION
**Pull request by eugene-chow**

### CODE REVIEW

**Comment by eugene-chow** [comment]
@aledbf please review. thanks!

File: keepalived-vip/README.md [hunk 1]
```diff
@@ -6,38 +6,81 @@ AKA "how to set up virtual IP addresses in kubernetes using [IPVS - The Linux Vi
 
 ## Overview
 
-There are 2 ways to expose a service in the current kubernetes service model:
+kubernetes v1.6 offers 3 ways to expose a deployment:
```
Comment (bowei) [inline keepalived-vip/README.md]: deployment -> service

File: keepalived-vip/README.md [hunk 2]
```diff
@@ -6,38 +6,81 @@ AKA "how to set up virtual IP addresses in kubernetes using [IPVS - The Linux Vi
...
 
-- Create a cloud load balancer.
-- Allocate a port (the same port) on every node in your cluster and proxy traffic through that port to the endpoints.
+1. **LoadBalancer**: [Available only on cloud providers](https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/) such as GCE and AWS
+1. **Service**: The [NodePort directive](https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport) allocates a port on every worker node, which proxy the traffic to the respective Pod.
```
Comment (bowei) [inline keepalived-vip/README.md]: Service -> Via NodePort

File: keepalived-vip/README.md [hunk 3]
```diff
@@ -6,38 +6,81 @@ AKA "how to set up virtual IP addresses in kubernetes using [IPVS - The Linux Vi
 
 ## Overview
 
-There are 2 ways to expose a service in the current kubernetes service model:
+kubernetes v1.6 offers 3 ways to expose a deployment:
 
-- Create a cloud load balancer.
-- Allocate a port (the same port) on every node in your cluster and proxy traffic through that port to the endpoints.
+1. **LoadBalancer**: [Available only on cloud providers](https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/) such as GCE and AWS
```
Comment (bowei) [inline keepalived-vip/README.md]: L4 LoadBalancer

File: keepalived-vip/README.md [hunk 4]
```diff
@@ -6,38 +6,81 @@ AKA "how to set up virtual IP addresses in kubernetes using [IPVS - The Linux Vi
...
+1. **Service**: The [NodePort directive](https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport) allocates a port on every worker node, which proxy the traffic to the respective Pod.
+1. **Ingress**: The [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) (a beta feature) is a dedicated loadbalancer (eg. nginx, HAProxy, traefik, vulcand) that redirects incoming HTTP/HTTPS traffic to the respective endpoints
```
Comment (bowei) [inline keepalived-vip/README.md]: Ingress -> L7 Ingress
Remove (a beta feature) [reduces doc churn when ingress goes GA]

**Comment by eugene-chow** [comment]
@bowei Updated the docs as requested. Thanks!

**Review by bowei — APPROVED** [review]

**Comment by eugene-chow** [comment]
\\o//

**Pull request merged**